content/panorama/scripts/team_select.js

```diff
@@ -30,6 +30,7 @@ function hidePregameUI () {
 function showPregameUI () {
   FindDotaHudElement('PreGame').style.opacity = 1;
   FindDotaHudElement('PreGame').style.visibility = 'visible';
+  FindDotaHudElement('PreGame').FindChildTraverse( 'GameModeLabel' ).text = "Open Angel Arena";
```

Comment:
I think it should remain, because we can write the result of voting for lenght there, I just have to find an output of it, WIP right now